feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/vif.c

```diff
@@ -1338,7 +1372,6 @@ static int ap_vif_config_set(const struct schema_Wifi_Radio_Config *rconf,
 		if (vconf->ft_psk && vconf->ft_mobility_domain) {
```

Comment:
You have no change here BUT, for the EAP case, shouldn't we expect vconf->ft_psk =0 and vconf->ft_mobility_domain is a value.  If this were so, then this config will not get applied.

Also this code sets ft_psk_generate_local which then gets rewritten later in vif_config_security_set().  This currently happens in the correct order but seems to be asking for confusion later.

Finally, this answers my earlier question; ft_mobility_domain is consumed her.